server/src/utils.ts

```diff
@@ -1,24 +1,25 @@
 //export const jwt = require('jsonwebtoken');
-import jwt from 'jsonwebtoken'
+import { User } from '@prisma/client';
+import jwt, { JwtPayload } from 'jsonwebtoken'
 export const APP_SECRET = 'GraphQL-is-aw3some';
 
-export const getTokenPayload = (token) => {
+export const getTokenPayload = (token:string) => {
```

Comment:
Both for JWT and Req/Response, you can use **type casting** as a (bit) better solution than just `any`: https://stackoverflow.com/questions/50735675/typescript-jwt-verify-cannot-access-data